config/locales/dashboard.en.yml

```diff
@@ -50,6 +50,59 @@ en:
       type: 'Type'
       actions: 'Actions'
 
+    ###################
+    #  FEE AGREEMENT  #
+    ###################
+    fee_agreement:
+      title: "Consolidated Fee Agreement"
+      settings:
+        header: "Settings"
+        report_configuration: "Report Configuration"
+        max_columns_per_table: "Max Visit Columns Per Table"
+        filters: "Filters"
+        status: "Status"
+        program: "Program"
+      study_info:
+        irb: "IRB"
+        date: "Date"
+        primary_investigator: "Primary Investigator"
+        sponsor_name: "Sponsor Name"
+        funding_source: "Funding Source"
+      arms:
+        header: "Arms"
+        total_count: "Total count of Arms: %{count}"
+        name: "Arm Name"
+        enrollment: "Enrollment"
+      budget_summary:
+        header: "Budget Summary"
+        total_non_clinical_services_cost: "Total Non-Clinical Services Cost"
+        total_clinical_services_cost: "Total Clinical Services Cost"
+        total_study_budget: "Total Study Budget"
+      non_clinical_services:
+        header: "Non-Clinical Services and Initiation Fees"
+        program: "Program"
+        service_name: "Service Name"
+        your_cost: "Your Cost"
+        quantity: "Quantity"
+        total: "Total"
+      clinical_services:
+        header: "Clinical Services"
+        program: "Program"
+        service_name: "Service Name"
+        service_cost: "Service Cost"
+        unit: "Unit"
+        enrollment: "Enrollment"
+        per_service_total: "Per Service Total"
+        service_notes: "Service Notes"
+        program_per_patient: "%{name} (Per Patient)"
+        total_visit_cost_per_patient: "Total Visit Cost (Per Patient)"
+        arm_total: "Arm Total: %{total}"
+      terms:
+        header: "Terms and Conditions"
+        content: "<li>This fee covers the administrative and regulatory review for all protocols using CTRC services.</li>
+          <li>Prices are subject to change due to hospital and institution rate adjustments.</li>
+          <li>Cost estimates are subject to revision periodically and with any change in services requested. If this is a cost estimate for a grant please request an updated cost estimate for every resubmission as rates may change.</li>
+          <li>When budgeting for your project, please include a rate increase each calendar year. Also note that institution costs are subject to F&A.</li>"
```

Comment:
These terms are not exactly what we use in the OCTRI translation file. I removed information specific to our organization (percentages and institution names). I'm open to other language if you have some default text we should use.